client/js/components/statement/listStatements/ListStatements.vue

```diff
@@ -487,7 +487,7 @@ export default {
     items () {
       return Object.values(this.statementsObject)
         .map(statement => {
-          const segmentsCount = statement.relationships.segments.data.length
+          const segmentsCount = statement.attributes.segmentsCount
```

Comment:
nitpick; I guess the default should be 0?
```suggestion
          const { segmentsCount = 0 } = statement.attributes
```

Reply:
[57b6221](https://github.com/demos-europe/demosplan-core/pull/5266/commits/57b6221d31d2d923f79045af7613434b4bf452c8)